src/main/resources/reference.conf

```diff
@@ -37,4 +37,15 @@ hmrc-timeout-dialog {
   defaultCountdownInSeconds = 120
 }
 
+pta-account-menu {
+  pertax-frontend {
+    host = "http://localhost:9232"
+    host = ${?platform.frontend.host}
+  }
+  accountHomePath = "/personal-account"
+  messagesPath = "/personal-account/messages"
+  checkProgressPath = "/track"
+  yourProfilePath = "/personal-account/your-profile"
```

Comment:
Is business-account URL part of another ticket?

Reply:
Yes, its part of [PLATUI-1468](https://jira.tools.tax.service.gov.uk/browse/PLATUI-1468) which we will be looking at next.